config/config-tracing.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-tracing
+  namespace: knative-serving
+data:
+  _example: |
+    ################################
+    #                              #
+    #    EXAMPLE CONFIGURATION     #
+    #                              #
+    ################################
+
+    # This block is not actually functional configuration,
+    # but serves to illustrate the available configuration
+    # options and document them in a way that is accessible
+    # to users that `kubectl edit` this config map.
+    #
+    # These sample configuration options may be copied out of
+    # this block and unindented to actually change the configuration.
+    #
+    # If true we enable adding spans within our applications.
+    enable: "false"
+
+    # URL to zipkin collector where traces are sent.
+    zipkin-collector-endpoint: "http://zipkin.istio-system.svc.cluster.local:9411/api/v2/spans"
+
+    # Enable zipkin debug mode. This allows all spans to be sent to the server
+    # bypassing sampling.
+    debug: "false"
+
+    # Percentage (0-1) of requests to trace
+    sample-rate: 0.1
```

Comment:
Do we want a new configmap, or should this be a part of config-observability?

Reply:
I could be swayed either way, but config-observability seems to contain mostly monitoring config which is a separate optional subsystem/set of components. My hope was to make it clear this is something baked in to knative-serving.

Reply:
Yeah, at the some time, I question whether some of this could be useful for tracing through system components in eventing 🤔   cc @Harwayne @grantr @mdemirhan

Reply:
cc @n3wscott @srinivashegde86 who are both working on tracing IIUC